docs/tuf-spec.txt

```diff
@@ -1004,8 +1028,8 @@ Version 1.0 (Draft)
    required, the root.json file is versioned and accessible by version number,
    e.g. 3.root.json. Clients update the set of trusted root keys by requesting
    the current root.json and all previous root.json versions, until one is
-   found that has been signed by keys the client already trusts. This is to 
-   ensure that outdated clients remain able to update, without requiring all 
+   found that has been signed by keys the client already trusts. This is to
```

Comment:
Thanks is now given (in the PR summary) to Evan and the CoreOS team.  In a separate edit to the specification, a "contributions" section on features that have been contributed by others will be added to the specification.

I think thanks is also given to the author(s) in the pull request that implements the feature.